api/src/YoutubeClient.cpp

```diff
@@ -90,6 +94,21 @@ rapidjson::Document VideosResult::GetSchema() {
                     }
                   },
                   "required": ["viewCount"]
+                },
+                "contentDetails: {
+                  "type": "object",
+                  "properties": {
+                    "contentRating": {
+                      "type": "object",
+                      "properties: {
+                        "ytRating": {
+                          "type": "string",
+                          "pattern": "^[0-9]+$"
+                        }
```

Comment:
`ytRating` should be required

Reply:
Can these actually be required since the API doesn't return these reliably?

Reply:
do we need to explicitly request these fields from the yt api here https://github.com/MemeLabs/Rustla2/blob/master/api/src/YoutubeClient.cpp#L148 ?

Reply:
either the values need to be required or accessing them needs to be checked with `HasMember`... i've been calling this KeyExists by mistake... either way is fine as long as we're getting the values when they're available from the api